extensions/smallrye-jwt/deployment/src/main/java/io/quarkus/smallrye/jwt/deployment/SmallRyeJwtProcessor.java

```diff
@@ -47,7 +69,7 @@ void registerAdditionalBeans(BuildProducer<AdditionalBeanBuildItem> additionalBe
         removable.addBeanClass(JsonValueProducer.class);
         removable.addBeanClass(JwtPrincipalProducer.class);
         additionalBeans.produce(removable.build());
-
+        additionalBeans.produce(new AdditionalBeanBuildItem(Claim.class));
```

Comment:
`AdditionalBeanBuildItem` is actually removable by default, so you can just add `Claim` to the `removable` build item above...

Reply:
Fixed